src/components/Card/Card.stories.tsx

```diff
@@ -63,17 +86,26 @@ function CardHeaderTemplateWithAdornment() {
 function CardContentTemplate() {
   return (
     <CardContent>
-      <Paragraph margin="none">
+      <Text size="md">
```

Comment:
The `Paragraph` component here was correct. `Text` is rendering span by default. It does not matter that much in this case but we should give correct examples to others.